README.md

```diff
@@ -1,54 +1,33 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/iwf799us8e4sjk4x/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/rhinoceros_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Rhinoceros_Toolkit/Rhinoceros_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=93&branchName=master)
 
-A great place to start is reading our Wiki [here](https://github.com/BHoM/documentation/wiki) including pages like the [Structure of the BHoM](https://github.com/BHoM/documentation/wiki/Structure-of-the-BHoM) and [Using the BHoM](https://github.com/BHoM/documentation/wiki/Using-the-BHoM).
+# Rhinoceros_Toolkit
 
-## Quick start ##
+This toolkit allows interoperability between the BHoM and McNeel Rhinoceros3d. Enables creation, conversion, manipulation and reading of Rhinoceros geometrical objects.
 
-Try the [installer](http://bhom.xyz/assets/installers/v2.1/BHoM%20Alpha%20v2.1.0.5%20Installer.exe) and a selection of [sample scripts](https://github.com/BHoM/samples).
+https://www.rhino3d.com/
 
+### Known Versions of Software Supported
+- Rhinoceros v5  
+- Rhinoceros v6
 
+### Documentation
+For more information about functionality see https://github.com/BHoM/Rhinoceros_Toolkit/wiki/BHoM---Rhinoceros-conversion-table
```

Comment:
Can you make this a proper link like see [here](https://github.com/BHoM/Rhinoceros_Toolkit/wiki/BHoM---Rhinoceros-conversion-table) or see [https://github.com/BHoM/Rhinoceros_Toolkit/wiki/BHoM---Rhinoceros-conversion-table](https://github.com/BHoM/Rhinoceros_Toolkit/wiki/BHoM---Rhinoceros-conversion-table)